installer/PowerToysSetup/Product.wxs

```diff
@@ -254,6 +254,75 @@
             <RegistryValue Type="string" Value="{0440049F-D1DC-4E46-B27B-98393D79486B}"/>
         </RegistryKey>
       </Component>
+      <Component Id="Module_PowerPreview" Guid="FF1700D5-1B07-4E07-9A62-4D206645EEA9" Win64="yes">
```

Comment:
We should start adding comments to easily identify sections of the file for specific modules.

Reply:
Added comments to describe the component and `dll's` being included.